tsconfig.json

```diff
@@ -6,11 +6,8 @@
     "declaration": true,
     "emitDeclarationOnly": true,
     "exactOptionalPropertyTypes": true,
-    "forceConsistentCasingInFileNames": true,
```

Comment:
this is now the default https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#deprecations-and-default-changes